packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -250,8 +253,26 @@ export const CommandMenu = () => {
     callback: closeCommandMenu,
   });
 
-  const selectableItemIds = matchingCreateCommand
-    .map((cmd) => cmd.id)
+  const isAskAIEnabled = useIsFeatureEnabled('IS_ASK_AI_ENABLED');
+  const setAskAIQuery = useSetRecoilState(askAIQueryState);
+  const openAskAIRightDrawer = useOpenAskAIRightDrawer();
+
+  const askAiCommand: Command = {
+    id: 'ask-ai',
+    to: '', // TODO
+    Icon: IconSparkles,
+    label: 'Ask AI',
```

Comment:
Didn't look into the code but it seems it's the only command where Enter key doesn't work?

Reply:
Fixed.